spec/requests/loan_documents_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "LoanDocuments", type: :request do
+  describe "GET /edit" do
+    it "returns http success" do
+      get "/loan_documents/edit"
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET /show" do
```

Comment:
@ericksonmc this one won't be needed. also, we can skip the request tests, they won't cover anything that controllers test check. But we do require controller tests.

Every change needs to be reflected at:
- model/service testing
- controller testing: all pathways possible for the action
- feature testing: happy path workflow